lib/bot-service.js

```diff
@@ -26,7 +26,19 @@ botService.get = async (resource) => {
             });
             response.on('end', () => {
                 const data = decodeToken(body);
-                data === undefined ? reject() : resolve(data);
+                let responseBody = {};
+                //Handle error from decoding token
+                if (data instanceof Error)  {
+                  reject(data)
+                }
+                //Handle error from Server
+                if (response.statusCode > 299){
```

Comment:
there is a space missing. But you could use an autoformatter or linter to automatically fix this! Again not a dealbreaker but for instance WebStorm you can easily avoid these things.

Reply:
Fixed it..